### ISSUE TITLE
Accessibility link

### ISSUE THREAD

**Issue by Dean-A-Smith** [alphagov/govuk-design-system-backlog#203]
## What

Establish how best to name and link to information about the accessibility options available to users within services. 

Is referring to 'Ease of access' (or similar) more likely to be recognised by users who could benefit from the information provided?

## Why

As part of meeting the legal requirements for the accessibility, teams producing websites and applications are directed to [publish an accessibility statement](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps)

Within this, services can give useful information to users about the controls available to the users, such as browser options to control text size and colours, certain assistive tools and so on.

Most services seem to do this through a link named 'Accessibility' page in the footer.

However, in a [Slack discussion in the cross-government 'Accessibility' channel](https://ukgovernmentdigital.slack.com/archives/C28AJTE07/p1571239323088400), members of the community shared examples where users: 

- do not recognise the term 'Accessibility'
- do not think that it is relevant to them
- do not bother to look because, from past experience, they expect it to contain technical information or jargon that isn't relevant

This means some users can miss out on information about ease of access that would make it easier for them to use a service. 

## Quotes from Slack discussion (as of now)

Leon Hubert, a user researcher at GDS, said:
[I've seen] "all of the above (when researching how users perceived accessibility statements for the GDS accessibility team). Particularly that users don't always associate the word with their needs, potentially consider it to be about physical access, don't expect it to contain anything useful for them, or are so used to those pages containing generic/jargony info that they don't bother to look."

Veronika Jermolina, a user researcher at Ministry of Justice, said:
"I’ve tested accessibility pages in the past. In usability testing users didn’t recognise the link and when prompted to look under it, didn’t think the information was relevant or helpful.
In the past, accessibility pages were largely a box-ticking exercise and had little valuable info for users. They usually consisted of a standard line about WCAG compliance (even if it was not true), HTML validation and browser keyboard shortcuts. The better ones linked to BBC’s 'my web, my way'.
I expect that even those who had looked there, over time learned not to waste their time.
Having observed recent discussions about accessibility statements, I think there’s still a risk of tick-boxing. That’s why any sample statements should include a massive caveat that they shouldn’t be copied and pasted and that you should first do the work, then write about it (in that order).
I think it would be interesting to get quant data on this with analytics."

Anike Henke, Front-End Developer and Accessibility Specialist at GDS, said:
"Bad discoverability was one of the reasons why Microsoft renamed their accessibility settings to “Ease of access”." 
(But I don't know the source for this)

In the 'Apply for a Budgeting Loan' service, feedback about the usability of the service is broadly very positive but a small percentage of users consistently give feedback that they need larger text or for text to be read to them. For this service we know some of our user base struggle in terms of literacy and can be low in confidence and ability using technology. This service doesn't currently have an accessibility statement of its own but it seems that users have not read GOV.UK's accessibility statement about the options available to them.

**Comment by 36degrees** [comment]
Thanks, Dean. I’ve removed the names of individuals from the post above – please be careful when adding things to the backlog not to include personally identifiable information.

**Comment by titlescreen** [comment]
On the DfE claim project, we expected people to view our accessibility statement when something wasn't working as expected. As such, we included mentions of known issues and workarounds (such as pasting into dragon pad when using dragon naturally speaking). 

Several users who identified themselves as having dyslexia mentioned that they would sometimes zoom the page in and one or two said they may change try and change the colour of a webpage., all things that are not possible to do without an app or browser extension.

https://www.claim-additional-teaching-payment.service.gov.uk/student-loans/accessibility-statement

On a side note, a discussion with a user who had lost their vision highlighted the need to follow convention. They were looking for a contact link in the footer but at the time we had an email link. Needless to say, they could not find it even when we told them there was an 'email link' in the footer.
